docs/Makefile

```diff
@@ -57,3 +57,13 @@ ${MDGEN_BINARY}: $(shell find ${CORSO_LOCAL_PATH}/src -type f -name *.go) $(shel
 clean:
 	$(DOCSC) rm -rf docs/cli build node_modules
 	$(GOC) rm -rf ${CORSO_BUILD_DIR}/*
+
+publish: clean build
+	docker run -e AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY \
+		-e AWS_SESSION_TOKEN -e AWS_REGION \
+		--rm -v ${PWD}:/usr/src/docs corso/docs:latest \
+	make sync
+
+sync:
+	aws s3 sync /usr/src/docs/build/ s3://corsobackup.io/ --exclude ".git/*" --delete
+	aws cloudfront create-invalidation --distribution-id E1W9NGI9YTVZ1A --paths "/*"
```

Comment:
@meain - should update docs publish off main (CD) to use the makefile and also override the bucket/CF with test values. Not a blocker for this PR since the docs job doesn't use this yet